### ISSUE TITLE
why dice loss?

### ISSUE THREAD

**Issue by BowieHsu** [argman/EAST#68]
Hi, I found loss function in model.py is dice classification loss which is different from the origin paper's balanced cross-entropy，does there exists paper provide the algorithm about dice classification? Thanks.

**Comment by BYJRK** [comment]
I have the same question. By the way, the Dice loss (1-2*IoU) looks kind of like the balancing factor beta in the class-balanced cross-entropy formula.

**Comment by BowieHsu** [comment]
@BYJRK  I read this paper “**Generalised Dice overlap as a deep learning loss function for highly unbalanced segmentations**” which compared dice loss and Weighted cross-entropy, the paper result shows that dice loss on segmentation task maybe more robust, i tried dice loss on my own task and found it gets better performance.

**Comment by BYJRK** [comment]
@BowieHsu Thanks a lot! This is really helpful.
